.travis.yml

```diff
@@ -96,17 +96,48 @@ matrix:
       script:
         - "[[ $APPIMAGE_BUILD_ARCHS == *armhf* ]] && ./build/travis/job2_AppImage/build.sh --armhf --upload-branches $APPIMAGE_UPLOAD_BRANCHES || true"
 
-notifications:
-  recipients:
-    - $NOTIFICATION_EMAIL
-  email:
-    on_success: change
-    on_failure: always
-  irc:
-    channels:
-      - $NOTIFICATION_IRC
-    on_success: change
-    on_failure: change
-  webhooks:
-    # trigger Buildtime Trend Service to parse Travis CI log
-    - https://buildtimetrend.herokuapp.com/travis
+
+before_install:
+  - export CC=/usr/bin/gcc-4.9
+  - export CXX=/usr/bin/g++-4.9
+  - "mkdir qt5"
+  - "wget -q -O qt5.zip http://utils.musescore.org.s3.amazonaws.com/qt542.zip"
+  - "unzip -qq qt5.zip -d qt5"
+  - "export PATH=`pwd`/qt5/bin:$PATH"
+  - "export QT_PLUGIN_PATH=`pwd`/qt5/plugins"
+  - "export QML2_IMPORT_PATH=`pwd`/qt5/qml"
+before_script:
+  #compatibility between ruby travis artefact and GO one
+  - "export ARTIFACTS_KEY=$ARTIFACTS_AWS_ACCESS_KEY_ID"
+  - "export ARTIFACTS_SECRET=$ARTIFACTS_AWS_SECRET_ACCESS_KEY"
+  - "artifacts -v || curl -sL https://raw.githubusercontent.com/meatballhat/artifacts/master/install | bash"
+  - "mkdir $HOME/software"
+  - "export PATH=$PATH:$HOME/bin:$HOME/software/bin"
+  - "make revision"
+  - "make debug CPUS=2 PREFIX=$HOME/software"
+  - "make installdebug CPUS=2 PREFIX=$HOME/software"
+  - "cd build.debug/mtest"
+  - "make -j2"
+script:
+  - "../../build/run_tests.sh"
+after_script:
+  - "export TRAVIS_SHORT_COMMIT=`echo $TRAVIS_COMMIT | cut -c 1-8`"
+  - "export ARTIFACTS_TARGET_PATHS=$TRAVIS_SHORT_COMMIT"
+  - export ARTIFACTS_PATHS=$(ls ../../vtest/html | tr "\n" ":")
+  - "export ARTIFACTS_PERMISSIONS=public-read"
+  - "export TRAVIS_BUILD_DIR=../../vtest/html"
+  - "artifacts upload"
+#notifications:
+#  recipients:
+#    - build@musescore.org
+#  email:
+#    on_success: change
+#    on_failure: always
+#  irc:
+#    channels:
+#      - "irc.freenode.net#musescore"
+#    on_success: change
+#    on_failure: change
+#  webhooks:
+#    # trigger Buildtime Trend Service to parse Travis CI log
+#    - https://buildtimetrend.herokuapp.com/travis
```

Comment:
Looks like you commented out these lines to prevent notification emails being sent. This is no longer necessary because the [email address is stored in the Travis environment](https://github.com/musescore/MuseScore/pull/2588/files#diff-354f30a63fb0907d4ad57269548329e3L101) so it won't be set on your Travis and is hidden on a PR. Same for the [IRC channel](https://github.com/musescore/MuseScore/pull/2588/files#diff-354f30a63fb0907d4ad57269548329e3L107). In fact, you can probably just checkout a fresh copy of .travis.yml from upstream and that might help with the AppImage compilation.